packages/taquito-michel-codec/test/global-constant.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { Parser, ParserOptions } from '../src/micheline-parser';
+import {
+  globalConstant,
+  globalConstantJSON,
+  script,
+  scriptJSON,
+} from './helpers/global-constants-helpers';
+
+describe('Expand global constants', () => {
+  const registeredIntExprJSON = { prim: 'int' };
+  const registeredDropExprJSON = { prim: 'DROP', args: [{ int: '2' }] };
+
+  const parserOptions: ParserOptions = {
+    expandGlobalConstant: {
+      constantHashInt: registeredIntExprJSON,
+      constantHashDrop: registeredDropExprJSON,
+    },
+  };
+
+  const p = new Parser(parserOptions);
+
+  it('Expands global constants in script using parseJSON', () => {
```

Comment:
this might just be my experience as an SDET speaking, but I think the way our describe blocks are written it's better to use the 'Should...' model of writing test descriptions.

So it will be written like 'Should expand global constants in script using parseJSON'. Seeing that most of our tests already have that description format, is that something we are willing to continue adopting?

Reply:
Good catch, I made the changes. Better to be consistent.